### DISCUSSION TITLE
umi 打包的问题

### DISCUSSION THREAD

**Discussion by old-zsh** [umijs/umi#9945] [Q&A]
我用umi打包，打包出来有两个js，一个umi.js,一个134.js。umi.js 中引用了134.js。我有两个问题
1 我将dist目录放置cdn上，umi.js是加cdn的域名地址的。为什么134.js没有加cdn的地址，现在导致，浏览器找不到134.js。报错，这个问题怎么解决
[](url)
![w1](https://user-images.githubusercontent.com/20679549/205483864-b3e48c2e-95b7-4ec4-b1c6-712fae7338c0.
![w2](https://user-images.githubusercontent.com/20679549/205483872-cfe427d8-dff8-444e-b014-2f42a14fc398.png)
png)

**Comment by old-zsh** [comment]
![w1](https://user-images.githubusercontent.com/20679549/205483916-4fed8486-83d4-4343-a314-fe634747dcdf.png)

**Comment by fz6m** [comment]
需要配置 [`publicPath`](https://umijs.org/docs/api/config#publicpath) 来确定加载资源的位置。

**Reply by old-zsh** [reply]
怎么配置呢

**Reply by old-zsh** [reply]
> 需要配置 [`publicPath`](https://umijs.org/docs/api/config#publicpath) 来确定加载资源的位置。

咋配置呢，帮忙看一下

**Reply by fz6m** [reply]
```ts
// .umirc.ts

export default {
  publicPath: process.env.NODE_ENV === 'production' ? '//cdn.domain.com/subpath/' : '/'
}
```

从而资源将从 `cdn.domain.com/subpath/umi.js` / `cdn.domain.com/subpath/123.js` 加载。

**Reply by old-zsh** [reply]
> 需要配置 [`publicPath`](https://umijs.org/docs/api/config#publicpath) 来确定加载资源的位置。

这个134.js 是动态import进来的一个包，然后umi给打包成了一个134.js。请问，可以将这个动态js打包到umi.js里面吗。或者告诉我一下，我这种情况怎么配置publicPath
